apps/jenkins/jenkins/sbox-intsvc/jenkins-azure-vm-agent.yaml

```diff
@@ -27,76 +96,16 @@ spec:
                         value: "sandbox"
                     deploymentTimeout: 1200
                     existingResourceGroupName: "aks-infra-cftsbox-intsvc-rg"
-                    maxVirtualMachinesLimit: 10
```

Comment:
you may need to set this to 0 otherwise the previous limit will still apply I think

Reply:
done